docker-compose.yml

```diff
@@ -1,56 +1,60 @@
 version: "3.9"
 services:
-  dashboard:
-    image: louislam/uptime-kuma:1
-    volumes:
-      - /var/run/docker.sock:/var/run/docker.sock
-      - ./uptime-kuma-data:/app/data
-    ports:
-      - "3001:3001"
-    restart: unless-stopped
-
   ddd-autohaus:
     image: ghcr.io/dqualizer/dqsamples-ddd-autohaus
+    container_name: 01_sample-application
     ports:
-      - "9000:9000"
+      - "${SAMPLE_APPLICATION_PORT}:${SAMPLE_APPLICATION_PORT}"
```

Comment:
Mapping to the same variable should be avoided, imo. Having an env-variable in such case does not have much sense as it can not be changed without breaking the compose file. Also check other occurrences.

Either have two different variables or only one for external. The internal port is (in any case) only used at this line in the compose and does, therefore, not need an env-variable for reuse at other places.